packages/react-kitchen-sink/src/react-query/querySnapshotQueryOptions.ts

```
@@ -1,18 +1,18 @@
 import { type DocumentData, type Query as FirestoreQuery, type SnapshotListenOptions } from '@firebase/firestore'
+import { type DefaultError, type QueryKey } from '@tanstack/react-query'
```

ESLint

[error] Unable to resolve path to module '@tanstack/react-query'.

(import-x/no-unresolved)